README.md

```diff
@@ -1,4 +1,30 @@
+
 # Tacoma Web Design and SEO
+
 ## Web Design Starter Template
 
-This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
\ No newline at end of file
+  
+
+This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
+
+## Optimization Changes
+
+
+- [Auto Image Optimization](#auto-image-optimization)
+- Auto Generate Critical CSS
+- Auto Sitemap Generator
+- Auto Robots Generator
+- CSS and JS Minify
+- CSS Auto Prefixer
+- Netlify Plugin Cache
+- [Local HTML Validation](#local-html-validation)
+
+
+### Auto Image Optimization
+ `data-img2picture-ignore` - add this in the `img` tag for icons, like star, phone, pin or any small icons.
```

Comment:
can you add more of a description to these. Also I think we should come up with a better name. If this is just for icons then we should have icons in the name

Reply:
Add additional info for this btw, also we can't really change the `data-img2picture-ignore` name as it's based on the package we use https://github.com/saneef/eleventy-plugin-img2picture